tests/cephfs/cephfs_multifs/multifs_split_merge.py

```diff
@@ -0,0 +1,271 @@
+import timeit
+import traceback
+
+from ceph.ceph import CommandFailed
+from ceph.parallel import parallel
+from tests.cephfs.cephfs_utils import FsUtils
+from tests.cephfs.cephfs_utilsV1 import FsUtils as FsUtilsV1
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    CEPH-11233 - Subtree Split and Subtree merging by pinning Subtrees/directories to MDS.
+    Args:
+        ceph_cluster:
+        **kw:
+    Steps Performed:
+    1. Create FS and mount on all the ways (Fuse, kernel, NFS)
+    2. Write Data to the mounts
+    3. Create directory and pin it to rank 0
+    4. Enable allow_dir_fragmentation flag and set max_mds to 2
+    5. Stress IO and check if the load is shared upon 2 MDS using get subtrees
+
+    Returns:
+        0 --> if test PASS
+        1 --> if test FAIL
+
+    """
+    try:
+        start = timeit.default_timer()
+        tc = "11232 and 11233"
```

Comment:
Since we have merged TC 11232 and 11233, please remove the inactive TC reference here.